src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -74,6 +74,9 @@
     @Argument(fullName = "use_jdk_inflater", shortName = "jdk_inflater", doc = "Whether to use the JdkInflater (as opposed to IntelInflater)", common=true)
     public boolean useJdkInflater = false;
 
+    @Argument(fullName = "nio_max_retries", shortName = "nio_retries", doc = "If the NIO channel errors out, how many times it will attempt to retry", optional = true)
+    public int NIO_MAX_REOPENS = BucketUtils.DEFAULT_NIO_MAX_REOPENS;
```

Comment:
Rename this to `gcs_max_retries` and explain in the doc that it applies to Google Cloud Storage requests -- the term "NIO" is meaningless to our users.